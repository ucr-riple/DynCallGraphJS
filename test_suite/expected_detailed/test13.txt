{
    'global (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test13.js:3:1:3:28)': [
        'call (Native)(/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test13.js:3:1:3:28)'
    ]
}