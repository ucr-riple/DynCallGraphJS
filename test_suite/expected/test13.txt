{
  '(test13.js:3:1:3:28)': [ 'call (Native)' ]
}